nemo/collections/nlp/models/language_modeling/megatron/gpt_full_te_layer_autocast_spec.py

```diff
@@ -125,6 +125,7 @@ def __init__(
         if te_version > packaging.version.Version("1.5.0"):
             transformer_layer_args["ub_overlap_ag"] = kwargs.get("ub_overlap_ag", True)
             transformer_layer_args["ub_overlap_rs"] = kwargs.get("ub_overlap_rs", True)
+            transformer_layer_args["ub_overlap_rs_dgrad"] = kwargs.get("ub_overlap_rs_dgrad", True)
```

Comment:
This version check ensures backward compatibility with TE <=1.5, but it doesn't handle the case where Mcore is old and TE is >1.5.